New Tutorial: Detection of Mitoflashes (#5514)
galaxyproject/training-material: topics/imaging/tutorials/detection-of-mitoflashes/tutorial.md

>
>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
>
> 3. Rename the datasets to keep track of them easily, e.g., "MitoFlash."
> 4. Confirm that the datatypes are correct for each file:
>    - `Mitoflash_CM-8bit.tif` should be an image file format.
>
>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
> 5. Tag each dataset with a label such as "mitoflash" for easy identification later if you are working multiple images.
>
{: .hands_on}

# Analyzing Mitochondrial Movement

In this section, we will focus on identifying mitochondrial regions in a time-lapse sequence by detecting spots based on local intensity maxima. This step provides insights into mitochondrial energy production and cellular health dynamics.

## Step 1: Detecting Mitochondrial Regions

> <hands-on-title>Detecting Mitochondrial Regions</hands-on-title>
>
> 1. {% tool [Spot Detection](toolshed.g2.bx.psu.edu/repos/imgteam/spot_detection/ip_spot_detection/1.0.0) %} with the following parameters:
>    - {% icon param-file %} *"Image input"*: `Mitochondrial_Flashes.tiff`

> >    - Set parameters such as "Thresholding" to optimize mitochondrial detection.

Reviewer: I think it would be beneficial for a tutorial if the instructions on such aspects were more explicit (e.g., suggest some good values).

Besides, it looks really good! 👏

Mitoflashes are triggered by elevated **reactive oxygen species (ROS)** levels and various physiological and stress-related signals including fluctuations in mitochondrial membrane potential, transient openings of the mitochondrial permeability transition pore (mPTP), calcium level changes, and bioenergetic stress (like nutrient deprivation or high metabolic demand). Additionally, they can be induced during specific cellular processes, like differentiation and certain developmental stages, where mitochondrial function is tightly regulated. This burst event activates the mitochondrial permeability transition pore (mPTP), causing a depolarization of the mitochondrial membrane potential (ΔΨm) and a mild alkalization of the matrix. While mitoflashes signal mitochondrial activity and ROS changes, they do not precisely quantify ROS levels or other cellular signatures. These events, however, are able to distinguish active from static mitochondria, making them valuable biomarkers for tracking mitochondrial health and function.

> The frequency and kinetics of mitoflashes hold significant physiological and pathophysiological implications. They correlate with key processes such as muscle contraction, cell differentiation, neuron development, wound healing, and lifespan prediction. The frequency and characteristics of mitoflashes vary by cell type; for example, adult cardiomyocytes display approximately 3.8 ± 0.5 mitoflashes, while primary cultured hippocampal neurons show around 31 ± 4 per cell.

Author: I added them =), thank you!

In this section, we will focus on identifying mitochondrial regions in a time-lapse sequence by detecting spots based on local intensity maxima. This step provides insights into mitochondrial energy production and cellular health dynamics.

## Step 1: Detecting Mitochondrial Regions

> <hands-on-title>Detecting Mitochondrial Regions</hands-on-title>
>
> 1. {% tool [Spot Detection](toolshed.g2.bx.psu.edu/repos/imgteam/spot_detection_2d/ip_spot_detection_2d/0.0.1) %} with the following recommended parameters:
>    - {% icon param-file %} *"Image input"*: `Mitochondrial_Flashes.tiff`
>    - **Starting time point** (`frame_1st`): `1` (first frame in the stack)
>    - **Ending time point** (`frame_end`): `0` (use `0` to process until the last frame)
>    - **Intensity measurement method** (`typ_intens`): Set to `Smoothed` for a more averaged intensity or `Robust` for resilience to outliers.
>    - **Threshold** (`thres`): A starting value of `10%` of the global maximum intensity is often effective, but increasing to `15-20%` is possible for higher sensitivity.
>    - **Gaussian filter sigma** (`ssig`): Use a value around `1` to suppress noise; increase to `1.5` if noise is high.

> >    - **Boundary pixels** (`bndy`): Set to `10` to ignore spots within the image edges, adjusting as needed for image size.

Author: Thank you! And yes, you are correct about the threshold, I corrected it ;)

>    This tool detects mitochondria as spots or 'mitoflashes' based on intensity maxima, identifying them as distinct regions. Adjusting the threshold parameter can improve the accuracy of spot detection, allowing for better identification of mitoflash events.
>
{: .hands_on}

> ## Step 2: Tracking Mitochondrial Movement

Reviewer: I think it would be good to give a brief motivation here for why tracking of the mitochondria is required in order to analyze their intensity changes over time.

> <hands-on-title>Tracking Mitochondrial Movement</hands-on-title>
>
> 1. {% tool [Perform linking in time series (nearest neighbors)](toolshed.g2.bx.psu.edu/repos/imgteam/points_association_nn/ip_points_association_nn/0.0.3-2) %} with the following recommended parameters:
>    - {% icon param-file %} *"Coordinates (and intensities) of input points"*: Output from the **Spot Detection** tool.
>    - **Neighborhood size** (`nbpx`): Set to `6` pixels as a starting value. Adjust within `1-10` pixels based on the density of points and proximity between spots across frames.
>    - **Intensity threshold** (`thres`): A recommended starting value is `25%` of the global maximum intensity. Increase to `30-40%` for stricter filtering if spots have low intensity.
>    - **Minimum track length** (`minlen`): Use `50%` of the sequence length as a baseline. Adjust based on the expected duration of mitoflash events.
>    

> >    This tool links detected spots across frames to track movement. The **Neighborhood size** parameter controls the pixel range for associating points between frames. Increasing the **Intensity threshold** filters out weaker signals, improving track accuracy.

Reviewer: Due to the [description of the relevant input of the tool](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/imgteam/points_association_nn/ip_points_association_nn/0.0.3-2),

> Tracks with all intensities lower than certain percentage (%) of the global maximal intensity will be discarded")

I suggest the following change:
```suggestion
>    This tool links detected spots across frames to track movement. The **Neighborhood size** parameter controls the pixel range for associating points between frames. Reducing the **Intensity threshold** filters out weaker signals, reducing the number of spurious tracks.
```

>
{: .hands_on}

# Detecting Mitoflashes

Mitoflashes are identified based on sudden changes in fluorescence intensity in mitochondria, signifying superoxide bursts. We will use a curve-fitting tool to capture these fluctuations and measure their characteristics.

## Step 3: Curve Fitting for Mitoflash Detection

> <hands-on-title>Mitoflash Detection</hands-on-title>
>
> 1. {% tool [Perform curve fitting](toolshed.g2.bx.psu.edu/repos/imgteam/curve_fitting/ip_curve_fitting/0.0.3-2) %} with the following parameters:

> >    - {% icon param-file %} *"File name of input data points (xlsx)"*: Select `spots_linked.xlsx`, the output file from the **Perform linking in time series (nearest neighbors)** tool.

Reviewer: I think the file produced by the preceding tool does not have the name `spots_linked.xlsx` in general, so either you should describe that the output should be renamed, or simply write it analogously to how you did for the previous step:
```suggestion
>    - {% icon param-file %} *"File name of input data points (xlsx)"*: Output from the **Perform linking in time series (nearest neighbors)** tool.
```

>    - **Degree of the polynomial function**: Set to `2nd degree` to capture intensity peaks that characterize mitoflash events.
>    - **Penalty**: Choose *Least absolute deviations (LAD)* for robust fitting to intensity fluctuations.
>    - **Alpha**: Set a significance level, such as `0.01`, to generate assistive curves if needed.
>

> >    This step applies polynomial curve fitting to model intensity fluctuations over time that represent mitoflash events. After running the tool, key parameters to analyze from the output file include:

Reviewer: I think it is important to be clear here. I added some explanation to improve clarity. Maybe you can try to be even a bit more clearer on the role of the "assistive curves" here:
```suggestion
>    This step applies polynomial curve fitting to obtain a model of the intensity fluctuations over time. Values which exceed the curve model by the given significance level, are recognized as mitoflash events. After running the tool, key parameters to analyze from the output file include:
```

> >    This step applies polynomial curve fitting to model intensity fluctuations over time that represent mitoflash events. After running the tool, key parameters to analyze from the output file include:

Reviewer: It would be very helpful if you could include the table produced by the tool for the example data you used for this tutorial here, before describing the columns. Like maybe a screenshot. Or maybe also just a section of the table.

Then you could also say how to read the table, for the specific example.

>
{: .hands_on}

# Detecting Mitoflashes

Mitoflashes are identified based on sudden changes in fluorescence intensity in mitochondria, signifying superoxide bursts. We will use a curve-fitting tool to capture these fluctuations and measure their characteristics.

## Step 3: Curve Fitting for Mitoflash Detection

> <hands-on-title>Mitoflash Detection</hands-on-title>
>
> 1. {% tool [Perform curve fitting](toolshed.g2.bx.psu.edu/repos/imgteam/curve_fitting/ip_curve_fitting/0.0.3-2) %} with the following parameters:
>    - {% icon param-file %} *"File name of input data points (xlsx)"*: Output from the **Perform linking in time series (nearest neighbors)** tool.
>    - **Degree of the polynomial function**: Set to `2nd degree` to capture intensity peaks that characterize mitoflash events.
>    - **Penalty**: Choose *Least absolute deviations (LAD)* for robust fitting to intensity fluctuations.
>    - **Alpha**: Set a significance level, such as `0.01`, to generate assistive curves if needed.
>

> >    This step applies polynomial curve fitting to model intensity fluctuations over time. The chosen **Penalty** used for curve fitting ensures that the curve is not affected by strong but short intensity deviations (e.g., mitoflashes or image noise) but rather represents the basic underlying intensity patterns. Mitoflash events can then be found by looking for intensity values which exceed the fitted curve by the given significance level. To facilitate this by thresholding, an assistive curve is generated which that corresponds to the significance level. Intensity values above the thresholds given by the assistive curve exceed the given significance level and correspond to mitoflash events.

Reviewer: Sorry I typo of mine ;)
```suggestion
>    This step applies polynomial curve fitting to model intensity fluctuations over time. The chosen **Penalty** used for curve fitting ensures that the curve is not affected by strong but short intensity deviations (e.g., mitoflashes or image noise) but rather represents the basic underlying intensity patterns. Mitoflash events can then be found by looking for intensity values which exceed the fitted curve by the given significance level. To facilitate this by thresholding, an assistive curve is generated that corresponds to the significance level. Intensity values above the thresholds given by the assistive curve exceed the given significance level and correspond to mitoflash events.
```

Mitoflashes are triggered by elevated **reactive oxygen species (ROS)** levels, and various physiological and stress-related signals including fluctuations in mitochondrial membrane potential, transient openings of the mitochondrial permeability transition pore (mPTP), calcium level changes, and bioenergetic stress (like nutrient deprivation or high metabolic demand).{% raw %}`{% cite Wang2022 %}`{% endraw %} Additionally, they can be induced during specific cellular processes, like differentiation and certain developmental stages, where mitochondrial function is tightly regulated. This burst event activates the mPTP, causing a depolarization of the mitochondrial membrane potential (ΔΨm) and a mild alkalization of the matrix.{% raw %}`{% cite Feng2019 %}`{% endraw %} While mitoflashes signal mitochondrial activity and ROS changes, they do not precisely quantify ROS levels or other cellular signatures. These events, however, are able to distinguish active from static mitochondria, making them valuable biomarkers for tracking mitochondrial health and function.

> Importantly, the frequency and kinetics of mitoflashes hold physiological and pathophysiological implications. They correlate with key processes such as muscle contraction, cell differentiation, neuron development, wound healing, and lifespan prediction. The frequency and characteristics of mitoflashes vary by cell type; for example, adult cardiomyocytes display approximately 3.8 ± 0.5 mitoflashes, while primary cultured hippocampal neurons show around 31 ± 4 per cell.{% raw %}`{% cite Li2020 %}`{% endraw %}

Author: So, I am used to adding the citation at the end of the sentence, after the period. This is a very common style in the field I come from—medical sciences. The citation would look like this: Myocardial infarction is a worldwide health concern.¹

We keep yours, it is okay with me, and I think it is more consistent with GTA. Thanks for making the changes!

>
{: .hands_on}

# Detecting Mitoflashes

Mitoflashes are identified based on sudden changes in fluorescence intensity in mitochondria, signifying superoxide bursts. We will use a curve-fitting tool to capture these fluctuations and measure their characteristics.

## Step 3: Curve Fitting for Mitoflash Detection

> <hands-on-title>Mitoflash Detection</hands-on-title>
>
> 1. {% tool [Perform curve fitting](toolshed.g2.bx.psu.edu/repos/imgteam/curve_fitting/ip_curve_fitting/0.0.3-2) %} with the following parameters:
>    - {% icon param-file %} *"File name of input data points (xlsx)"*: Output from the **Perform linking in time series (nearest neighbors)** tool.
>    - **Degree of the polynomial function**: Set to `2nd degree` to capture intensity peaks that characterize mitoflash events.
>    - **Penalty**: Choose *Least absolute deviations (LAD)* for robust fitting to intensity fluctuations.
>    - **Alpha**: Set a significance level, such as `0.01`, to generate assistive curves if needed.
>

> >    This step applies polynomial curve fitting to model intensity fluctuations over time, identifying mitoflash events when intensity values exceed the fitted curve by the given significance level. Key parameters to analyze are **Amplitude** (peak intensity), **Tpk** (time to peak), and **T50** (duration). These values provide initial insights into mitochondrial function, with further interpretation outlined in the following section.

Reviewer: Not quite clear what the parameters Amplitude, T50, and Tpk refer to. The tool doesn't have those parameters?

Author: Thank you for asking! These parameters—**Amplitude**, **Tpk**, and **T50**—are not input parameters of the tool but are derived metrics used to measure mitoflash events. In biology, they are commonly reported to quantify and analyze mitochondrial dynamics. 

For example:
- **Amplitude** measures the peak intensity of the flash.
- **Tpk** reflects the time it takes to reach this peak.
- **T50** indicates the duration of the flash.

These values are useful for plotting and interpreting mitoflash dynamics in biological studies. Perhaps it would be more accurate to call these *"metrics that can be extracted from the output for plotting"*? 

Several papers mention these metrics; for example, [this one](https://pmc.ncbi.nlm.nih.gov/articles/PMC7580095/).

Rather than being annotations, these metrics are quantitative descriptors that help summarize and interpret key features of mitoflash events. Please let me know what you think and if this helps the clarification. I wanted to add this info to guide the user to the analysis and interpretation part.

Reviewer: I see, thanks for the clarification! I think this hasn't been sufficiently clear from the tutorial :) I think it would be better to put this into the section after the last step of the workflow, as this seems to be like a description of "further analysis" beyond the workflow. So it would be more natural to first finish the description of the workflow, then describe the further analysis.

> These values are useful for plotting and interpreting mitoflash dynamics in biological studies. Perhaps it would be more accurate to call these "metrics that can be extracted from the output for plotting"?

To further avoid confusion, I think it would be better to avoid the term "parameter" here if possible, "metrics" sounds much better.

> >    This step applies polynomial curve fitting to model intensity fluctuations over time, identifying mitoflash events when intensity values exceed the fitted curve by the given significance level. Key parameters to analyze are **Amplitude** (peak intensity), **Tpk** (time to peak), and **T50** (duration). These values provide initial insights into mitochondrial function, with further interpretation outlined in the following section.

Reviewer: The important text part from the paper that explains the curves:
> To better distinguish mitoflashes, we use a second-degree polynomial curve […] to fit the intensities of one mitochondrion over time […] For curve fitting, we [use] the least absolute deviations (LAD) which is more robust to noise and outliers. **This means that the fitted curves will be much less affected by mitoflashes and well represent the basic intensity change patterns**, making the detection of mitoflashes easier and more robust.

And for the assistive curves:
> To objectively quantify significant deviations, for each image sequence we compute the distributions of the measured mitochondria intensities deviating from the fitted curves and **determine a threshold that corresponds to a significance level of 0.01. An intensity increase above this threshold is regarded as "significant," which is used as a base criterion for distinguishing mitoflash events.**

Based on this information, I suggest the following changes:
```suggestion
>    This step applies polynomial curve fitting to model intensity fluctuations over time. The chosen **Penalty** used for curve fitting ensures that the curve is not affected by strong but short intensity deviations (e.g., mitoflashes or image noise) but rather represents the basic underlying intensity patterns. Mitoflash events can then be found by looking for intensity values which exceed the fitted curve by the given significance level. To facilitate this by thresholding, an assistive curve is generated which that corresponds to the significance level. Intensity values above the thresholds given by the assistive curve exceed the given significance level and correspond to mitoflash events.
```

>
{: .hands_on}

## Analysis and Interpretation of Results

Finally, now you have your results table ready for analysis! The output is an Excel file containing the columns **FRAME**, **POS_X**, **POS_Y**, **INTENSITY**, **CURVE**, and **CURVE_A**. This information is available for all detected spots in your image stack. 

With our data results, we can analyze their frequency, duration, and intensity. This data helps us understand the physiological significance of mitoflashes in processes like muscle contraction, neuron development, and wound healing.

> ![Curve fitting results mitoflash](../../images/detection-of-mitoflashes/Curve_fitting_results_mitoflash.png "Mitoflash Curve Fitting Results: Data for plotting.")

Reviewer: Very cool! Now this might be somewhat picky of me, but if you have the time, I think it would be better to show the table for a different spot (I suspect that there is a spot for which the INTENSITY value somewhere exceeds the CURVE_A value, is there?)

Author: I will find it and add it =D! Maybe we can add a question here for it then? - Hahah more work

### Key Parameters for Analysis and Plotting

1. **FRAME**: Represents time points in the data, with each row corresponding to a frame in the time-lapse sequence. Use **FRAME** as the x-axis for time-based plotting.

> 2. **POS_X** and **POS_Y**: These columns provide positional coordinates of each detected mitoflash, allowing visualization of mitochondrial movement and spatial distribution within the cell.

Reviewer: ```suggestion
2. **POS_X** and **POS_Y**: These columns provide the coordinates of each detected mitoflash, allowing visualization of mitochondrial movement and spatial distribution within the cell.
```

3. **INTENSITY**: Shows the raw intensity values for each mitochondrion at each time point. Plotting **INTENSITY** against **FRAME** reveals fluctuations over time, with peaks indicating mitoflash events.

4. **CURVE** and **CURVE_A**: Contain fitted curve data that smooths out noise to highlight true intensity peaks. **CURVE** is the main fit, while **CURVE_A** may show adjusted fits if available.
   
   - **Amplitude (F/F0)**: The peak value in **CURVE** or **CURVE_A**, representing the magnitude of each mitoflash. Higher amplitudes indicate stronger bursts, which can have biological significance.
   - **Tpk (Time to Peak)**: The frame where **CURVE** or **CURVE_A** reaches maximum intensity, indicating the speed of each mitoflash.

>    - **T50 (Duration)**: The frames where the curve values are at least half of the peak value, representing the mitoflash duration. Longer durations may reflect sustained mitochondrial activity.

Reviewer: I think this needs to be adapted to be more consistent with the description of the curves above (see my comments for line 136). Where do those terms Amplitude, Tpk, T50 come from? I couldn't find them in the curve fitting tool.

---
layout: tutorial_hands_on

title: Tracking of mitochondria and capturing mitoflashes
zenodo_link: https://zenodo.org/records/14071552
questions:
- What are mitoflashes, and why are they relevant for understanding mitochondrial function?
- How can bioinformatics and image analysis tools help in tracking and analyzing mitochondrial dynamics?
objectives:
- Understand the biological significance of mitoflashes and their implications for cellular health.
- Learn to track mitochondrial movements in live-cell imaging data.
- Use image analysis tools to detect and quantify mitoflashes.
time_estimation: 1H
key_points:
- Mitoflashes are superoxide events in mitochondria that signal functional states and stress responses.
- Bioinformatics tools can facilitate the visualization, tracking, and quantitative analysis of these events.
- Proper tracking and analysis of mitoflashes contribute to understanding mitochondrial behavior and health in cells.

contributions:
  authorship:
    - dianichj
    - kostrykin
tags:
  - bioimaging
  - mitoflash
  - mitochondria
    
---

> # Introduction

Reviewer 2: @bgruening if you're reviewing things in the future, could we trouble you to please ask folks to remove the '# introduction' section header, they should just start the text, rather than including this header.

Reviewer 3: Ok, I will try to remember. It does not look that bad :)

I fixed it in Leonids PR.

Importantly, the frequency and kinetics of mitoflashes hold physiological and pathophysiological implications. They correlate with key processes such as muscle contraction, cell differentiation, neuron development, wound healing, and lifespan prediction. The frequency and characteristics of mitoflashes vary by cell type; for example, adult cardiomyocytes display approximately 3.8 ± 0.5 mitoflashes, while primary cultured hippocampal neurons show around 31 ± 4 per cell ({% cite Li2020 %}).

In this tutorial, you will learn to track mitochondria in live-cell imaging data and detect mitoflashes using Galaxy tools for image analysis. You will work with time-lapse microscopy data, often stored as TIFF files with image stacks, to observe and analyze mitochondrial events across multiple time points. By identifying these events and quantifying their frequency through intensity measurements fitted to a curve, you'll gain insights into mitochondrial behavior and activity over time.

> This tutorial is based on a workflow originally described in % cite Wang2022 %.

Author: Oh Sorry! I missed it, thanks for adding it 🙏😁

>
{: .agenda}

# Preparing Your Data

First, we need to upload the data we'll work with. Ensure that you have the necessary files prepared from Zenodo or a shared data library within Galaxy.

## Data Upload

> <hands-on-title>Data Upload</hands-on-title>
>
> 1. Create a new history for this tutorial in Galaxy.
>
>    {% snippet faqs/galaxy/histories_create_new.md %}
>
> 2. Import the mitoflash imaging data from [Zenodo](https://zenodo.org/records/14071552) or from the shared data library:
>    - **Important:** Choose the correct data type if prompted (tiff).
>
>    ```
>    https://zenodo.org/records/14071552/files/mitoflashes_8bit.tiff
>    ```
>
>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
>    
>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
> 
> 3. Confirm that the datatypes are correct for each file:
>    - `mitoflashes_8bit.tiff` should be an image in **tiff** file format.
>
>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
> 4. Tag each dataset with a label like "mitoflash_1" for easy identification later if you are working with multiple images.
>
{: .hands_on}

# Analyzing Mitochondrial Movement

> In this section, we will focus on identifying mitochondrial regions in a time-lapse sequence by detecting spots based on local intensity maxima. This step provides insights into mitochondrial energy production and cellular health dynamics.

Author: Thanks for pointing that out! You’re absolutely right—at this stage, we’re just detecting spots that represent local intensity maxima, which might indicate mitochondrial activity but aren’t conclusive yet. The insights into energy production and cellular health come later, as we analyze and track these spots (like when identifying mitoflashes). I think your version works better for this step.

To answer your question: I haven’t worked directly with mitoflashes before, but based on the literature, they can indeed indicate shifts in energy production and cellular health. For example, if a cell is in a pathological state (such as a tissue under stress or injury), it may have a higher metabolic demand. This can lead to more superoxide bursts, which appear in the image as fluorescent signals, or "mitoflashes." These events often reflect increased energy demand (e.g., ATP production) or mitochondrial stress. So, if this big important organelle is stressed, the cell health state could be interpreted as compromised. 

As for the word "regions," it’s commonly used in image biology to refer to specific areas of interest within a structure, such as sections observed on a slide. These "regions" typically represent fluorescence from a compound binding to a specific site or component of the structure, rather than the entire organelle or cell. When I worked with slides of infarcted hearts, we even called them "zones." That might be why I referred to them as regions. However, in technical terms, they are essentially coordinates like you said, as the results reflect the intensity of a pixel at a specific (x,y) position, if I understand correctly.

> In this section, we will focus on identifying mitochondrial regions in a time-lapse sequence by detecting spots based on local intensity maxima. This step provides insights into mitochondrial energy production and cellular health dynamics.

Reviewer: Much thanks for the detailed clarifications!

I see the ambiguous use of the term "regions" in biological and image analysis context. I think it's still better to avoid that term due to this ambiguity, in case someone without biological background wants to follow the tutorial.

> ## Step 1: Detecting Mitochondrial Regions
> 
> > <hands-on-title>Detecting Mitochondrial Regions</hands-on-title>

Reviewer: You have used the term "regions" in the context of spot detection several times. I think this is misleading, from a technical point of view, because spot detection yields _points_ (coordinates), not _regions_. Determining image regions would rather require _segmentation_ instead of _detection_.

Due to this, I'd suggest the following changes, which also are more consistent with the figure that shows the workflow at the end of the tutorial:
```suggestion
## Step 1: Mitochondrion Detection

> <hands-on-title>Mitochondrion Detection</hands-on-title>
```
